lib/internal/crypto/keygen.js

```diff
@@ -60,8 +61,10 @@ const {
 const { isArrayBufferView } = require('internal/util/types');
 
 function wrapKey(key, ctor) {
-  if (typeof key === 'string' || isArrayBufferView(key))
-    return key;
+   if (typeof key === 'string' ||
+       isArrayBufferView(key) ||
+       isJwk(key))
+     return key;
```

Comment:
```suggestion
  if (typeof key === 'string' ||
      isArrayBufferView(key) ||
      isJwk(key))
    return key;
```
Lint issues caused by indentation.

Reply:
good